dockerfiles/Dockerfile_Linux.build.i386

```diff
@@ -0,0 +1,26 @@
+FROM i386/ubuntu:16.04
+
+# Update image
+SHELL ["/usr/bin/linux32", "/bin/bash", "-c"]
+
+RUN apt-get update && apt-get install -y sudo cmake build-essential curl libcurl4-openssl-dev \
+    libssl-dev uuid-dev python-pip apt-utils python-virtualenv python3
+
+WORKDIR /usr/sdk
+
+RUN python -m virtualenv --python=python3 env3
+RUN source env3/bin/activate && pip install --upgrade pip && pip install -U setuptools wheel
+
+# Copy code
```

Comment:
Recommend using more descriptive comment from some of other dockers, namely 
# Copy code (this assumes the ./src folder contains the recursively cloned SDK repository (azure/azure-iot-sdk-python))

Reply:
done